### PR TITLE
Api domain project

### PR DESCRIPTION
**Pull request by Goooler**
Follow up https://github.com/patrykmichalik/opto/issues/2#issuecomment-1232005066.

### CODE REVIEW

**Comment by patrykmichalik** [comment]
Thanks for the PR! Since you’ve expanded the README a bit, would it be all right with you if I added a couple more sentences to it by pushing to your branch?

**Comment by Goooler** [comment]
Fine, plz!

**Review by patrykmichalik — APPROVED** [review]

**Comment by patrykmichalik** [comment]
Thank you, @Goooler!

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
 ![](https://img.shields.io/maven-central/v/com.patrykmichalik.opto/core)
+
+Opto is a lightweight Preferences DataStore wrapper. Read more about it [here](https://patrykmichalik.com/projects/opto).
+
+## Getting started
+
+Opto is available on Maven Central. Ensure this repository is declared in your project-level `build.gradle` file.
+
+```groovy
+buildscript {
```
Comment (Goooler) [inline README.md:12]: Should be `allprojects` instead of `buildscript`.

Comment (Goooler) [reply README.md:12]: Found this error also in https://github.com/patrykandpatryk/vico#getting-started.

Comment (patrykmichalik) [reply README.md:12]: Ah, you’re right. I’ve corrected this both here and in Vico’s README. Thanks for letting me know about the error!

**Pull request merged**